### DISCUSSION TITLE
Tachometer connection question

### DISCUSSION THREAD

**Discussion by northmrytle** [hatlabs/discussions#58] [Q&A]
I am working on using the Sailor Hat with the Engine Top Hat to send engine data to a SignalK display and could use some assistance with a tachometer connection.

The engine is an older 12 volt gas V8 without a W terminal on the alternator that I can locate as described in this [discussion.](https://github.com/hatlabs/discussions/discussions/43) The existing tachometer gets its signal from the negative terminal of the ignition coil.

Am I overthinking this, can I simply connect a set of the digital pins to the coil positive and negative terminals to detect the opening and closing of the points or is there a circuit I can build to facilitate reading the tachometer signal from the coil ?

**Comment by mairas** [comment]
My understanding is that the ignition coil negative terminal alternates between 0 and 12V. If that's the case, then it's perfectly fine. Coil positive, I believe, is at 12V whenever ignition is turned on.

Do NOT connect the digital input ground pin to anything other than the common ground. If you connect it to the ignition coil negative terminal, you will burn the devices. You can leave the ground terminal disconnected, it should still work fine.
